gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -94,6 +93,10 @@ fluid.defaults("gpii.flowManager", {
             route: "/user/:userToken/logout",
             type: "get"
         },
+        userKeyaction: {
```

Comment:
Strange capitalisation here - see later notes on naming of "key"-related endpoints